tsdx.config.js

```diff
@@ -0,0 +1,13 @@
+const { nodeResolve } = require('@rollup/plugin-node-resolve')
+
+module.exports = {
+  rollup(config, options) {
+    config.plugins.push(
+      nodeResolve({
+        extensions: ['.js', '.json'],
+      })
+    )
+    
+    return config;
+  },
+};
```

Comment:
Nitpick: maybe a new line here. (I think this file hasn't been formatted with prettier)